pos_no_bank_statement/model/point_of_sale.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# This file is part of OpenERP. The COPYRIGHT file at the top level of
+# this module contains the full copyright notices and license terms.
+
+from openerp import models, api
+
+
+class POSOrder(models.Model):
+    _inherit = "pos.order"
+
+    @api.v7
```

Comment:
That decorator is used for that cases: you use cr, uid, context, and the third argument is a single id, loading in self the corresponding record, in contrast to `@api.multi`, that uses cr, uid, ids, context arguments to load a list of ids as the recordset.